specification/databricks/resource-manager/Microsoft.Databricks/preview/2023-09-15-preview/databricks.json

```diff
@@ -0,0 +1,1921 @@
+          "type": "string"
+        },
+        "workspaceUrl": {
+          "readOnly": true,
+          "description": "The workspace URL which is of the format 'adb-{workspaceId}.{random}.azuredatabricks.net'",
+          "type": "string"
+        },
+        "storageAccountIdentity": {
+          "description": "The details of Managed Identity of Storage Account",
+          "$ref": "#/definitions/ManagedIdentityConfiguration"
+        },
+        "managedDiskIdentity": {
+          "description": "The details of Managed Identity of Disk Encryption Set used for Managed Disk Encryption",
+          "$ref": "#/definitions/ManagedIdentityConfiguration"
+        },
+        "diskEncryptionSetId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource Id of the managed disk encryption set."
+        },
+        "encryption": {
+          "type": "object",
+          "description": "Encryption properties for databricks workspace",
+          "properties": {
+            "entities": {
+              "$ref": "#/definitions/EncryptionEntitiesDefinition",
+              "description": "Encryption entities definition for the workspace."
+            }
+          },
+          "required": [
+            "entities"
+          ]
+        },
+        "enhancedSecurityCompliance": {
+          "description": "Contains settings related to the Enhanced Security and Compliance Add-On.",
+          "$ref": "#/definitions/EnhancedSecurityComplianceDefinition"
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "description": "Private endpoint connections created on the workspace",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "description": "The network access type for accessing workspace. Set value to disabled to access workspace only via private link.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "PublicNetworkAccess"
+          }
+        },
+        "requiredNsgRules": {
+          "type": "string",
+          "description": "Gets or sets a value indicating whether data plane (clusters) to control plane communication happen over private endpoint. Supported values are 'AllRules' and 'NoAzureDatabricksRules'. 'NoAzureServiceRules' value is for internal use only.",
+          "enum": [
+            "AllRules",
+            "NoAzureDatabricksRules",
+            "NoAzureServiceRules"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "RequiredNsgRules"
+          }
+        },
+        "defaultCatalog": {
+          "description": "Information regarding the catalog associated with the workspace.",
```

Comment:
Use : "Information regarding the initial default catalog associated with the workspace." for description

Reply:
Updated the description